[2023-12-19T11:21:29.876+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-12-19T11:21:32.737+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.sense_json_file scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-12-19T11:21:32.816+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-19T11:21:35.421+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): sense_json_file> on 2023-01-03 00:00:00+00:00
[2023-12-19T11:21:35.716+0000] {standard_task_runner.py:57} INFO - Started process 163 to run task
[2023-12-19T11:21:35.972+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'sense_json_file', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/dag0001.py', '--cfg-path', '/tmp/tmpk6zs4wax']
[2023-12-19T11:21:36.281+0000] {standard_task_runner.py:85} INFO - Job 54: Subtask sense_json_file
[2023-12-19T11:23:02.916+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 54 for task sense_json_file ((psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: https://sqlalche.me/e/14/e3q8); 163)
[2023-12-19T11:23:14.197+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-12-19T11:23:15.737+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
